.markdownlint.yml

```diff
@@ -16,6 +16,7 @@ MD029: false     # Ordered list item prefix
 MD030: false     # Spaces after list markers
 MD033: false     # Allow inline HTML
 MD034: false     # Bare URL used
```

Comment:
This change is unrelated and should be removed from this PR

Reply:
I'm guessing it's related to updating markdownlint.

Reply:
yes, it's a new added rule that we were not following.

Reply:
Right. I’m saying such a change doesn’t belong in a PR that is just to upgrade dependencies.

Reply:
to not change the config, pushed a commit to fix the md037 issues: https://github.com/eslint/eslint/pull/14624/commits/b8b032a1ee7b573ab245354dcdce5e345cd709f5.

Reply:
I would actually recommend just ignoring `CHANGELOG.md` in a `.markdownlintignore` file like this: https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/.markdownlintignore

Reply:
sounds good!

Reply:
@aladdin-add it looks like this codebase is using markdownlint via CLI and also via its NodeJS API in `Makefile.js`. Only the CLI version will automatically consider the `.markdownlint` file. I think we should probably move this dependency upgrade to a separate PR since it is becoming more complicated. I can take that on if that sounds good.

Reply:
agreed to put in another PR! I will revert the markdownlint version.

Reply:
Separate PR to update markdownlint: https://github.com/eslint/eslint/pull/14883